spec/orm/activerecord_spec.rb

```diff
@@ -1956,6 +1960,15 @@ def reload(*)
       expect { @event.dup }.not_to change { @event[:image] }
     end
 
+    it "can upload a file" do
```

Comment:
```suggestion
    it "allows the original object to store a file" do
```

Can the title be a bit more descriptive like this?

Reply:
Thanks for your suggestion!